openstudiocore/resources/analysis/pat_export/scripts/StandardReports/measure.rb

```diff
@@ -1,139 +1,165 @@
 require 'erb'
+require 'json'
+
+require "#{File.dirname(__FILE__)}/resources/os_lib_reporting"
+require "#{File.dirname(__FILE__)}/resources/os_lib_schedules"
+require "#{File.dirname(__FILE__)}/resources/os_lib_helper_methods"
 
 #start the measure
-class StandardReports < OpenStudio::Ruleset::ReportingUserScript
+class OpenStudioResults < OpenStudio::Ruleset::ReportingUserScript
```

Comment:
Thanks, I didn't want to step on name of old one, so I just followed the name for E+ results. I also wanted the class name to match the display name, which wasn't true for the old measure; although on BCL it did snake_case the name when I uploaded it.